core/common-android/build.gradle.kts

```
@@ -0,0 +1,12 @@
+plugins {
+    alias(libs.plugins.team.noweekend.android.library)
+    alias(libs.plugins.team.noweekend.hilt)
+}
+
+android {
+    namespace = "team.noweekend.core.common.android"
+}
+
+dependencies {
+    implementation(project(":core:model"))
```

```suggestion
    implementation(project(":core:model"))
    implementation(project(":core:common-kotlin"))
```
p3 : common-kotlin 정도는 들어갈 수 있을 것 같아요

---

알겠습니다. 넣어놓도록 하져.